### DISCUSSION TITLE
The user case for the customizable web map

### DISCUSSION THREAD

**Discussion by dadiorchen** [Greenstand/treetracker-web-map-client#1675] [Ideas]
@RubenSmn @sebastiangaertner I am thinking about the use case for our customizable web map, maybe we should just make it available for the `embed` map mode, this is the major case for our partner to integrate our map to their website/mobile app, like this link: https://map.treetracker.org/organizations/1331?embed=true for example, the bondy organization, https://www.bondy.earth/en/#banner , use the embed map of bondy, and customize the map to fit their brand, color, spacing, we can make a good presentation of bondy application/website with Greenstand map for their product. 
On the other hand, the non-embed map https://map.treetracker.org/organizations/1331 it should be using the same style as others, because it's on the product `treetracker` we shouldn't change the style of the this link to the customized map by organizations, and if we keep changing the style when we jumping between organizations, it looks weird and disturbing for the user.

**Comment by dadiorchen** [comment]
<img width="1440" alt="image" src="https://github.com/Greenstand/treetracker-web-map-client/assets/5744708/58e5f248-913e-47b0-8d75-0aa8c74746f8">

**Comment by sebastiangaertner** [comment]
I think this is also a web wallet need, correct? Embedding isolated maps.

**Reply by dadiorchen** [reply]
@RubenSmn is it easy to extend to the wallet?

**Reply by RubenSmn** [reply]
I don't think that it will be much of a challenge, we can make the request to the server, get the theme data and force that theme on the web wallet pages. In the same way we're currently doing with the web map client.

**Reply by dadiorchen** [reply]
Can we say that we can proportionally change the spacing to some extends?  By changing the theme spacing size?

**Reply by RubenSmn** [reply]
Yes we can say that is possible :+1:
